source/main.css

```diff
@@ -0,0 +1,126 @@
+.info {
+    margin: 0px auto;
+    width: 80%;
+}
+
+div p {
+    text-align: left;
+    color: #000000;
+    font-size: 14px;
+    font-family: helvetica;
+}
+
+.photo-1 {
+    display: block;
+    width: 100%;
+    height: 900px;
+    /*margin: 0px auto;*/
+    background-image: url("./images/bb-photo1.png");
+    background-repeat: no-repeat;
+    background-position: center center;
+    background-size: cover;
+    text-align: center;
+}
+
+.bb-photo {
+    display: block;
+    width: 100%;
+    margin: 0px auto;
+}
+
+.in-text {
+    margin: 0px;
+}
+
+.second-nav {
+    height: auto;
+    width: 100%;
+    text-align: center;
+    background-color: #000000;
+}
+
+.social-home {
+    display: inline-block;
+    width: auto;
+    margin: 0px auto;
+}
+
+.social-link {
+    margin: auto 60px;
+    text-align: center;
+    font-size: 22px;
+}
+
+.foot-step {
+    border-bottom: 2px solid #000000;
+    margin: 5px auto;
+    width: 520px;
+}
+
+span {
+    color: #ff5d00;
+}
+
+p {
+    text-align: center;
+    font-family: Open Sans;
+    font-size: 12px;
+    margin-top: 0px;
+    color: #ff5d00;
+    margin-bottom: 45px;
+    text-decoration: bold;
+}
+
+nav {
+    text-align: center;
+}
+
+a {
+    margin: 0px;
+    font-size: 22px;
+    font-family: helvetica;
+    text-decoration: none;
+    color: black;
+}
+
+h1 {
+    text-align: center;
+    font-size: 44px;
+    font-family: Helvetica;
+    margin-bottom: 0px;
+    margin-top: 40px;
+}
+
+h2 {
+    float: right;
+    font-size: 34px;
+    padding-right: 20px;
+    margin: auto;
+    color: #ffffff;
+}
+
+h3 {
+    font-family: Helvetica;
+    font-size: 40px;
+    font-weight: lighter;
+    padding-left: 20px;
+    margin-bottom: 10px;
+    margin-top: 20px;
+}
+
+h5 {
+    display: inline-block;
+    text-align: center;
+    font-size: 70px;
+    color: #ffffff;
+    font-weight: lighter;
+    margin-top: 320px;
+}
+
+h6 {
+    margin: 0px;
+}
+
+footer {
+    background-color: pink;
```

Comment:
Avoid color names … I know this is just a place holder, so it's fine for now, but there's a saying in computer-land, "There's nothing more permanent than temporary."

Which just means it's easy to forget about this kind of stuff, and while this is pretty minimal it's a good habit to be pretty vigilant about isolating any "fake" or "temporary" stuff. If you remove the `footer` from the HTML, and forget to remove this style it'll just hang out here waiting to bite you in the ass.

Again, that's a little contrived. This site is super small, but on a bigger project it can be an issue. Maybe just inline-style any temporary HTML?